README.md

```diff
@@ -175,6 +176,16 @@ Example:
 
     wayback_machine_downloader http://example.com --concurrency 20
 
+## Specify UserAgent for connection
+
+     -u, --user-agent STRING
+
+UserAgent for connection (Default is WayBack Machine Downloader)
+
+Example:
+
+    wayback_machine_downloader http://example.com --user-agent "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:77.0) Gecko/20190101 Firefox/77.0"
```

Comment:
maybe use another UA than FF for the example, some smart tv UA's have long lifespans